app.js

```diff
@@ -1,83 +1,77 @@
+/*eslint no-undef: "error"*/
+/*eslint no-console: ["error", { allow: ["warn", "error"] }] */
+/*eslint-env node*/
+
 const express = require('express');
 const app = express();
-const morgan = require('morgan');
 const bodyParser = require('body-parser');
+const alasql = require('alasql');
+
 
 app.use(express.static('public'));
 
-const PORT = process.env.PORT || 4001;
-
-const items = [
-  {
-    id: 1,
-    name: 'Bread',
-    quantity: '1',
-    unit: 'Kg'
-  },
-  {
-    id: 2,
-    name: 'Milk',
-    quantity: '1',
-    unit: 'L'
-  },
-  {
-    id: 3,
-    name: 'Pasta',
-    quantity: '2',
-    unit: 'Kg'
-  }
-];
-let nextId = 4;
+const PORT = process.env.PORT || 4002;
+
+alasql('CREATE TABLE items (id int, name varchar(255), quantity int, unit varchar(25))');
+
+alasql('INSERT INTO items VALUES (\'1\',\'Bread\',\'1\',\'Kg\')');
+alasql('INSERT INTO items VALUES (\'2\',\'Rice\',\'2\',\'Kg\')');
+alasql('INSERT INTO items VALUES (\'3\',\'Pudding\',\'2\',\'Kg\')');
 
 // Body-parsing Middleware
 app.use(bodyParser.json());
 
-// Logging Middleware
-if (!process.env.IS_TEST_ENV) {
-  app.use(morgan('dev'));
-}
-
 // Get all Items
 app.get('/items/', (req, res, next) => {
```

Comment:
separate routes definition from app definition. something like:
in app.js:
```
app.use('/', require('./routes'))
```
in routes/index.js
```
const express = require('express')
const router = express.Router()
router.use('/endpoint1', .....)
module.exports = router

Reply:
Index.js created and code in app.js was reduced.